<?xml version="1.0"?>

<launch>
    <node name="path_planning" pkg="path_planning" type="path_planning.py" >
      <param name="chart_file" value="res_100_lon_-122.67_-122.22_lat_37.54_38.17.jpg" />
      <param name="goal_x" value="30000" />
      <param name="goal_y" value="-24980" />
    </node>

</launch>
